packages/pds/src/db/migrations/20230718T170914772Z-sequencer-leader-sequence.ts

```diff
@@ -0,0 +1,25 @@
+import { Kysely, sql } from 'kysely'
+import { Dialect } from '..'
+
+export async function up(db: Kysely<any>, dialect: Dialect): Promise<void> {
+  if (dialect === 'sqlite') return
+  const res = await db
+    .selectFrom('repo_seq')
+    .select('seq')
+    .where('seq', 'is not', null)
+    .orderBy('seq', 'desc')
+    .limit(1)
+    .executeTakeFirst()
+  const startAt = res?.seq ? res.seq + 10000 : 1
```

Comment:
opinion on doing this in the migration vs by hand?

there's some slight worry that during deploy the following happens:
- sequence is created in migration
- old leader steps down
- new leader (with new code) steps up
- new leader crashes/steps down for some reason
- old instance grabs leader  & throws off our bookkeeping

Reply:
I made a similar comment above, ha!  I agree, it might be smart to do manualy.  I also think that quick-reloads on the dedicated sequencer could help quite a bit.